Client/core/Graphics/CPrimitive3DBatcher.h

```diff
@@ -0,0 +1,39 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        CPrimitive3DBatcher.h
+ *  PURPOSE:
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#pragma once
+
+// Vertex type used by the primitives 3d batcher
+struct sPrimitive3D
+{
+    D3DPRIMITIVETYPE                 eType;
+    std::vector<PrimitiveVertice>*   pVecVertices;
+};
+
+//
+// Batches primitives drawing
+//
+class CPrimitive3DBatcher
+{
+public:
+    ZERO_ON_NEW
```

Comment:
@CrosRoad95 please replace ZERO_ON_NEW macro, refer to https://github.com/multitheftauto/mtasa-blue/pull/984#issuecomment-501146941